File: WeekOne/Password.js
//Made by Gustavoufjfjv1 - Gustavo Henrique

let caracteres = "1 2 3 4 5 6 7 8 9 0 q w e r t y u i o p a s d f g h j k l ç z x c v b n m Q W E R T Y U I O P A S D F G H J K L Ç Z X C V B N M ! @ # $ & * ? _ -"

let array = caracteres.split(' ')

let senha = "";

for(let i = 0; i < 8; i++){
  let novoCaractere;
  do {
    novoCaractere = array[Math.floor(Math.random() * array.length)];
  } while (senha.length > 0 && novoCaractere === senha[senha.length - 1]);
  senha += novoCaractere;
}

console.log(senha)

File: WeekOne/Password.cs
/* Made by RedicFrey  */

string Letters = "Q q W w E e R r T t Y y U u I i O o P p A a S s D d F f G g H h J j K l Z z X x C c V v B b N n M m 1 2 3 4 5 6 7 8 9 0 ! @ # $ % & * ? - _";
string[] LettersV = Letters.Split(" ");
string Password = "";
int V = 999;
Random r = new Random();


for (int i = 0; i < 10; i++)
{
    for (int j = 0; j < 8; j++)
    {
        int R = r.Next(0, 68);

        if (R != V)
        {
            Password += "" + LettersV[R];
        }
        else
        {
            j -= 1;
        }

        V = R;
    }

    Password += "\n\n";
}



Console.WriteLine(Password);
Console.ReadKey();
